runtime/src/memory.rs

```diff
@@ -61,7 +61,7 @@ impl PluginMemory {
 
     /// Write byte to memory
     pub(crate) fn store_u8(&mut self, offs: usize, data: u8) -> Result<(), MemoryAccessError> {
-        trace!("store_u8: {data:x} at offset {offs}");
+        trace!("store_u8: offset={offs} data={data:#04x}");
```

Comment:
it's `04` instead of `02` because rust includes the `0x` characters in the max width.